charts/external-dns-management/templates/deployment.yaml

```diff
@@ -917,7 +917,7 @@ spec:
         {{- end }}
       {{- end }}
       serviceAccountName: {{ include "external-dns-management.fullname" . }}
-      priorityClassName: {{ include "external-dns-management.fullname" . }}
+      priorityClassName: gardener-system-900
```

Comment:
We have several stakeholders using this helm chart to deploy the dns-controller-manager in their own cluster.
This change will break deployments outside of gardener as the priority class `gardener-system-900` is not existing in this case.